encoder/vaapiencoder_h264.cpp

```diff
@@ -220,10 +253,93 @@ bit_writer_write_trailing_bits(BitWriter *bitwriter)
 }
 
 static BOOL
-bit_writer_write_sps(
-    BitWriter* bitwriter,
-    const VAEncSequenceParameterBufferH264* const seq,
-    VideoProfile profile)
+bit_writer_write_sei(BitWriter* bitwriter,
+                     const VAEncSequenceParameterBufferH264* const seq,
+                     uint32_t temporalLayerNum)
+{
+    BitWriter scalabilityInfoWriter;
+    uint32_t i;
+
+    /* Write scalability_info */
+    scalabilityInfoWriter.writeBits(0, 1); // temporal_id_nesting_flag: false
+    scalabilityInfoWriter.writeBits(
+        0, 1); // priority_layer_info_present_flag: false
```

Comment:
Could we make scalabilityInfoWriter smaller, so we can get rid of this wired line breaking?